services/common/src/main/java/com/linkedin/openhouse/common/stats/model/IcebergTableStats.java

```diff
@@ -35,4 +36,6 @@ public class IcebergTableStats extends BaseTableMetadata {
   private Long numReferencedManifestFiles;
 
   private Long numReferencedManifestLists;
+
+  private Map<String, Long> snapshotCountByDay;
```

Comment:
if SE is functioning fine this should only have a bounded number of days?

Reply:
That is right. It should ideally have only 3 days worth of data. But we can consider collecting only past 2 days since we should already have the old data from previous runs